CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changes
 
 ## Next Release
+- Fixed logging problem related to django translation before logging ([#624](https://github.com/jazzband/django-auditlog/pull/624))
```

Comment:
```suggestion
#### Fixes

- Fixed logging problem related to django translation before logging ([#624](https://github.com/jazzband/django-auditlog/pull/624))
```

Reply:
I must have made the same mistake with the following change log : "feat: Excluding ip address when AUDITLOG_DISABLE_REMOTE_ADDR is set to True"

So that I can incorporate these changes as quickly as possible